packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -351,8 +362,13 @@ function ViewGrid< Item >( {
 										} }
 										aria-busy={ isLoading }
 										ref={ resizeObserverRef }
+										role={
+											isInfiniteScroll
+												? 'feed'
+												: undefined
+										}
```

Comment:
I'm really not sure if the semantics here are correct: when `groupByField` is enabled, this means there will be multiple `feed` role groups on the page. Should we just disable infinite scroll when grouping by field?

Reply:
Good question. I suppose semantically it'd be the containing `VStack` that would get the "feed" role if there should just be the one?

> Should we just disable infinite scroll when grouping by field?

From my perspective it would make sense to start out not supporting "group by" for now, and return to adding it further down the track if it's needed? Aside from the semantics of the `feed` role, it does feel like the group by case is a slightly different UX from scrolling through a single list 🤔